### PR TITLE
Upgrade to Cairo 2.7

### PR DESCRIPTION
**Pull request by piniom**

### CODE REVIEW

File: src/presets/user_account.cairo [hunk 1]
```diff
@@ -165,7 +165,7 @@ mod ArgentUserAccount {
             self: @ContractState,
             class_hash: felt252,
             contract_address_salt: felt252,
-            threshold: usize,
+            new_threshold: usize,
```
Comment (tarrencev) [inline src/presets/user_account.cairo:168]: Whats the reason behind this renaming?

File: src/recovery/interface.cairo [hunk 2]
```diff
@@ -57,7 +57,7 @@ enum EscapeStatus {
 /// @param ready_at when the escape can be completed
 /// @param target_signers the signers to escape
 /// @param new_signers the new signers to be set after the security period
-#[derive(Drop, Serde, starknet::StorePacking)]
```
Comment (tarrencev) [inline src/recovery/interface.cairo:60]: Is this default now?

File: src/presets/user_account.cairo (hunk 1 above)
Comment (piniom) [reply src/presets/user_account.cairo:168]: New cairo requires the signatures of the constructor and `__validate_deploy__` to be analogous. This is the name used in the constructor, so I've used it here as well.
```cairo
#[constructor]
fn constructor(ref self: ContractState, new_threshold: usize, signers: Array<Signer>) 
```

File: src/recovery/interface.cairo (hunk 2 above)
Comment (piniom) [reply src/recovery/interface.cairo:60]: No, I thought it's not supported any more with the general changes to the storage mechanism. It turns out the "packing" trait can still be implemented. This derive stopped working but I haven't found anything on why that's the case. I'll try to resolve it quickly.

**Pull request merged**